<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deanonwindow</class>
 <widget class="QMainWindow" name="deanonwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>258</width>
      <height>242</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="deanon_Lay">
     <item>
      <widget class="QLabel" name="deanonLab">
       <property name="text">
        <string>Choose De-anonymizing technique:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="deanonList"/>
     </item>
     <item>
      <widget class="QPushButton" name="deanonButt">
       <property name="text">
        <string>Choose</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="deanon_DesLay">
     <item>
      <widget class="QLabel" name="descriptLab">
       <property name="text">
        <string>Description of technique:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="descriptText"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>981</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="deanon_OutLay">
     <item>
      <widget class="QLabel" name="OutLab">
       <property name="text">
        <string>Output:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="OutTab"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>431</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="deanon_DispLay">
     <item>
      <widget class="QLabel" name="changeLab">
       <property name="text">
        <string>Label</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="changeText"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="backButt">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1176</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
